utils/src/field_helpers.rs

```diff
@@ -102,18 +124,21 @@ impl<F: Field> FieldHelpers<F> for F {
             bits
         })
     }
+
+    fn bits_to_field(&self, start: usize, end: usize) -> Result<F> {
+        F::from_bits(&self.to_bits()[start..end]).map_err(|_| FieldHelpersError::DeserializeBits)
+    }
 }
 
 /// Field element wrapper for [BigUint]
-pub trait FieldFromBig<F> {
-    /// Deserialize from big unsigned integer
-    fn from_biguint(big: BigUint) -> Result<F>;
+pub trait BigUintFieldHelpers<F: PrimeField> {
+    /// Convert BigUint into PrimeField element
```

Comment:
I thought perhaps you wanted a test using the to_field and the to_biguint to make sure they give you the same thing when applied on top of each other

Reply:
done